examples/http.glu

```diff
@@ -123,8 +123,8 @@ let catch_error action catch : Handler a -> (String -> Handler a) -> Handler a =
 /// Takes a `Handler` and a `Request` tries to process the request
 let handle handler state : Handler Response -> HttpState -> IO Response =
     let not_found _ _ =
-        pure { status = NotFound, body = "Page not found" }
-    handler (\response _ -> pure response) not_found state
+        wrap { status = NotFound, body = "Page not found" }
+    handler (\response _ -> wrap response) not_found state
```

Comment:
Yep, no worries, take your time!

I think the other nice thin is that it mirrors the `unwrap` from Rust, but I dunno... :/

I've been also thinking about how we could make extensible rows more useful using namespaced fields, but I'll save that for somewhere else!